rules/linux/lnx_process_discovery.yml

```diff
@@ -0,0 +1,22 @@
+title: Process Discovery
+id: 4e2f5868-08d4-413d-899f-dc2f1508627b
+status: stable
+description: Detects process discovery commands
+author: Ömer Günal, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1057/T1057.md
+logsource:
+    product: linux
+detection:
+    selection:
+        - ProcessName|contains:
+          - 'ps '
+          - 'top'
```

Comment:
Hello @omergunal! 

It's better to use `endswith` here and put a slash at the beginning of the name. 
also, space is redundant and could be deleted. 
you can use existing open PRs for Linux and macOS as a reference for the syntax (:

Reply:
thank you!